examples/backends/low_level/horqrux_dqc.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import annotations
+
+import jax
+import jax.numpy as jnp
+import matplotlib.pyplot as plt
+import optax
+from jax import Array, grad, jit, value_and_grad, vmap
+from numpy.random import uniform
+from numpy.typing import ArrayLike
+
+from qadence.backends import backend_factory
+from qadence.blocks.utils import chain
+from qadence.circuit import QuantumCircuit
+from qadence.constructors import feature_map, hea, ising_hamiltonian
+from qadence.types import BackendName, DiffMode
+
+N_QUBITS, DEPTH, LEARNING_RATE, N_POINTS = 4, 3, 0.01, 20
+# building the DQC model
+ansatz = hea(n_qubits=N_QUBITS, depth=DEPTH)
+# the input data is encoded via a feature map
+fm = feature_map(n_qubits=N_QUBITS, param="x", fm_type="chebyshev")
+# choosing a cost function
+obs = ising_hamiltonian(n_qubits=N_QUBITS)
+# building the circuit and the quantum model
+circ = QuantumCircuit(N_QUBITS, chain(fm, ansatz))
+bknd = backend_factory(BackendName.HORQRUX, DiffMode.AD)
+conv_circ, conv_obs, embedding_fn, params = bknd.convert(circ, obs)
+
+optimizer = optax.adam(learning_rate=0.001)
```

Comment:
Maybe still use the LEARNING_RATE defined above?

Reply:
added